tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+- name: Couchbase | Install Couchbase Repo
+  yum:
+    name: http://packages.couchbase.com/releases/couchbase-release/couchbase-release-1.0-0-x86_64.rpm
+    state: present
+
+- name: Couchbase | Install Couchbase
+  yum:
+    name: couchbase-server-4.5.0
```

Comment:
version should be configurable (I think there is a separate argument for version?)

Reply:
Yeah diff arg - will update to pass in :)